Yank/repex.py

```diff
@@ -1951,9 +1916,20 @@ def create(self, thermodynamic_states, sampler_states, storage, unsampled_thermo
                     raise ValueError('All ThermodynamicStates and SamplerStates must '
                                      'have the same number of particles')
 
+        # Handle case in which storage is a string.
+        reporter = self._reporter_from_storage(storage, check_exist=False)
```

Comment:
Could you add this? I'm not sure I understand the control flow well enough to tackle that.

Reply:
Sure. We don't need it for this PR though. I think it would be easier to put in after the initial SAMS implementation is in place. If you want though, I can do that now